### PR TITLE
refactor(readme): Update readme for LTS release 8-19-2020

### PR DESCRIPTION
**Pull request by abhipsaMisra**
@sharmasejal To confirm this - .NET SDK will now have a 1 year "active" maintenance period, and a 3 year "critical-fix" support period.

Before:
image

Now:
image

The readme is here: https://github.com/Azure/azure-iot-sdk-csharp/blob/abmisr/readme/readme.md

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -171,8 +171,8 @@ Below is a table showing the mapping of the LTS branches to the packages release
 
 | Release | Github Branch | LTS Status | LTS Start Date | Maintenance End Date | Removed Date |
 | :-----------: | :-----------: | :--------: | :------------: | :------------------: | :----------: |
-| [2020-1-31](https://github.com/Azure/azure-iot-sdk-csharp/tree/lts_2020-1-31_patch1) | lts_2020_01   | Active     | 2020-01-31     | 2020-06-30           | 2020-12-31   |
```
Comment (abhipsaMisra) [inline readme.md:174]: @sharmasejal Once we have confirmation on these timelines, could you share the pictorial representation that you have in your slides? I'd like to update the "Planned release schedule" image below.

File: readme.md [hunk 2]
```diff
@@ -171,8 +171,8 @@ Below is a table showing the mapping of the LTS branches to the packages release
```
Comment (abhipsaMisra) [inline readme.md:171]: We will also need to update the LTS description above:
A new LTS version will be created every **6 months**. The lifetime of an LTS branch is currently planned for **one year**. LTS branches receive all bug fixes that fall in one of these categories:

File: readme.md (hunk 1 above)
Comment (sharmasejal) [reply readme.md:174]: @elhorton For review.

**Comment by elhorton** [comment]
My general feedback is to add more context around this LTS change. Otherwise it may seem like we are just choosing arbitrary dates for these releases. I'm imagining something like: 

# Long Term Support Information
As of August 2020, the C# SDK is shifting to a revised Long-Term Support strategy. The primary motivations for this change are to extend the support period and decrease the churn on LTS releases, while still maintaining a strategy that offers customers choice between new features and stability. 

We now will be releasing a new LTS branch yearly, and each LTS release will be supported for 3 years-- 1 year of active maintenance with bugfixes, and 2 years of extended support for security fixes.  

image

## Schedule<sup>1</sup>

Below is a table showing the mapping of the LTS branches to the packages released

| Release | Github Branch | LTS Status | LTS Start Date | Maintenance End Date | Removed Date |
| :-----------: | :-----------: | :--------: | :------------: | :------------------: | :----------: |
| [2020-8-19](https://github.com/Azure/azure-iot-sdk-csharp/tree/lts_2020-8-19) | lts_2020-8-19   | Maintenance | 2020-08-19     | 2021-8-19           | 2023-8-19   |
| [2020-1-31](https://github.com/Azure/azure-iot-sdk-csharp/tree/lts_2020-1-31_patch1) | lts_2020_01   | Deprecated     | 2020-01-31     | 2020-06-30           | 2020-12-31   |

- <sup>1</sup> All scheduled dates are subject to change by the Azure IoT SDK team.

**Comment by abhipsaMisra** [comment]
Thanks for this @elhorton , this is exactly what I was looking for.

**Comment by abhipsaMisra** [comment]
Once this PR is merged into `master`, we will need to create a branch with the branch name `lts_2020_08`

**Pull request merged**